privacyidea/api/lib/prepolicy.py

```diff
@@ -1162,9 +1173,10 @@ def is_remote_user_allowed(req):
                                                          scope=SCOPE.WEBUI,
                                                          user=loginname,
                                                          realm=realm,
-                                                         client=g.client_ip)
+                                                         client=g.client_ip,
+                                                         audit_data=g.audit_object.audit_data)
 
-        res = ruser_active
+        res = ruser_active.keys()
```

Comment:
not sure what the calling function expects as return value but i guess it is a list.
Please use `res = list(ruser_active)`

Reply:
done in e6a3ae1

We now return  bool, since this is actually what the code expects.